script/Arcueid/Day 13/13_00_ARC13_6.txt

```diff
@@ -241,12 +241,13 @@ That I shouldn't look at death for too long. That if I push myself too hard, the
 [sha:fc850eb71b24899c8516ecf832f7254754d068aa]{
 -- Page 463, Offset 33010.
 -- こんなコトで良かったのなら。
-If I was okay with this...
+// not too sure on this change but my interpretation is that shiki assumes he's going to end up as a cripple hence why i worded it like this
+If I was prepared to end up like this...
```

Comment:
I assume this is like the big block in 9_2B. If that's the case, this is probably ok